docs/prometheus-integration.md

```diff
@@ -116,3 +116,165 @@ The configuration file above can be used to deploy Prometheus Server with
 scraping configuration for Antrea services.
 To deploy this configuration use
 `kubectl apply -f build/yamls/antrea-prometheus.yml`
+
+# Antrea Prometheus Metrics
```

Comment:
I don't know if this is the ideal place, but the doc should mention how devs who add metrics can re-generate the list.

Reply:
This doc is very user-oriented, while doc generation is targeted to developers.
As we have a bunch of other generatables, maybe it's worthwhile to create a dev-oriented code enlisting all of these (e.g yamls via make manifest, testing mocks etc). I can create an issue for this and then handle it separately.
BTW do we need a make target for the metrics generation as well?

Reply:
We do have a doc that may work already: https://github.com/vmware-tanzu/antrea/blob/master/docs/code-generation.md, although the title may need to be updated

I don't think a make target is needed right now, but we can add one in the future